src/move_basic.cpp

```diff
@@ -635,6 +647,17 @@ bool MoveBasic::rotate(double yaw, std::string drivingFrame)
         ros::spinOnce();
         r.sleep();
 
+        // Try to update goal - see comment in moveLinear()
+        tf2::Transform T_planning_driving;
+        tf2::Transform goalInDriving;
```

Comment:
Is this actually used anywhere in `rotate`? Doesn't seem like it is to me.

Reply:
Yes, 2 lines below.

Reply:
Yes but that just assigns to it, I don't see the value being used.

Reply:
Yes, I removed it.